include/rocksdb/flush_block_policy.h

```diff
@@ -45,9 +54,10 @@ class FlushBlockPolicyFactory {
 
 class FlushBlockBySizePolicyFactory : public FlushBlockPolicyFactory {
  public:
-  FlushBlockBySizePolicyFactory() {}
+  FlushBlockBySizePolicyFactory();
 
-  const char* Name() const override { return "FlushBlockBySizePolicyFactory"; }
+  const char* Name() const override { return kClassName(); }
```

Comment:
minor: change the order of "const char* Name()" and "static const char* kClassName()"